File: Examples/Ruby/managing-recognition/without-cloud-storage/post_barcode_recognize_from_request_body.rb
#ExStart:
require 'aspose_barcode_cloud'

class RecognizeBarcode

  include AsposeBarcodeCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("", "")
    @barcode_api = BarcodeApi.new
  end
  
  # Read Barcode from Request Body.
  def post_barcode_recognize_from_request_body
      file_name = "barcode.png"
      opts = {type: "Code39Standard", checksumValidation: "", stripFnc: false, rotationAngle: 0, file: File.open("../../../../data/" << file_name,"r") { |io| io.read }}
      response = @barcode_api.post_barcode_recognize_from_urlor_content(opts)
  end

end

recognizeBarcode = RecognizeBarcode.new()
puts recognizeBarcode.post_barcode_recognize_from_request_body
#ExEnd:

File: Examples/PHP/managing-recognition/without-cloud-storage/post-barcode-recognize-from-request-body.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/../..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/../..') . '/Utils.php';

use Aspose\Barcode\BarcodeApi;
use Aspose\Barcode\AsposeApp;

class RecognizeBarcode {

	public $barcodeApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->barcodeApi = new BarcodeApi();
	}

	public function postBarcodeRecognizeFromRequestBody() {

		$fileName = 'barcode.png';
        $file = realpath(__DIR__ . '/../../../..') . '/Data/' . $fileName;

        $type = 'Code39Standard';
        $checksumValidation = "";
        $stripFnc = false;
        $rotationAngle = 0;

        $result = $this->barcode->PostBarcodeRecognizeFromUrlorContent($type, $checksumValidation, $stripFnc, $rotationAngle, $url = null, $file);
		print_r($response);	
	}
}

$recognizeBarcode = new RecognizeBarcode();
$recognizeBarcode->postBarcodeRecognizeFromRequestBody();

?>
//ExEnd: